.circleci/config.yml

```diff
@@ -295,23 +285,46 @@ workflows:
   nightly:
     triggers:
       - schedule:
-          cron: "0 0 * * *"
+          cron: "0 23 * * *"
           filters:
             branches:
               only:
                 - staging
                 - main
+
     jobs:
-      - nightly-builds:
+      - run-task:
+          name: archive-build-logs-staging
+          app: federalistapp-staging
+          task-name: archive-build-logs-$(date "%Y-%m-%d")
+          task-command: yarn archive-build-logs-daily $(date "%Y-%m-%d")
```

Comment:
not sure if this date thing is gonig to work

Reply:
You could just retrieve the date in the function during execution since its only run once a day for that day.